packages/typespec-vscode/test/utils.ts

```diff
@@ -0,0 +1 @@
+export const it = window.test;
```

Comment:
Is this imported as a side-effect somewhere? Don't see this file being imported explicitly.

Reply:
nope I was trying to do something but I should remove, forgot